.github/workflows/build.yaml

```diff
@@ -11,17 +11,18 @@ jobs:
       - name: Checkout Packer project
         uses: actions/checkout@v2
 
-      - name: Packer Init
+      - name: Export Application Credentials
         run: |
-          curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
-          sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
-          sudo apt-get update && sudo apt-get install packer
-          packer init .
```

Comment:
Similar as in the other PR. Packer init should be added as a step, that will be executed prior to the `packer build`.